node-fs-writefile/cp.js

```diff
@@ -0,0 +1,13 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+async function cp() {
+  try {
+    const contents = await readFile(process.argv[2], 'utf8');
+    const copyFile = String(process.argv[3]);
+    await writeFile(copyFile, `${contents}\n`);
```

Comment:
I don't think you want to add an extra carriage return onto this file.

Reply:
Removed \n